tests/FSharp.Compiler.ComponentTests/EmittedIL/SteppingMatch/SteppingMatch07.fs.il.netcore.release.bsl

```diff
@@ -91,8 +91,9 @@
     .method assembly specialname rtspecialname 
             instance void  .ctor(int32 _tag) cil managed
     {
-      .custom instance void [runtime]System.Runtime.CompilerServices.CompilerGeneratedAttribute::.ctor() = ( 01 00 00 00 ) 

```

Comment:
Why is baseline updating dropping the CompilerGeneratedAttribute and DebuggerNonUserCodeAttribute attributes in most of the files?

I would have expected only an addition of DynamicDependencyAttribute.

Reply:
Yes we shouldn't be dropping the compiler generated attributes.  I will try to figure out why that is so.

Reply:
Fixed